src/Carable.AssemblyPayments/Implementations/ItemRepository.cs

```diff
@@ -239,7 +239,7 @@ public ItemRepository(IRestClient client, ILoggerFactory loggerFactory, IOptions
         public async Task<Item> ReleasePaymentAsync(string itemId, int releaseAmount)
         {
             AssertIdNotNull(itemId);
-            var request = new RestRequest("/items/:id/release_payment", Method.PATCH);
+            var request = new RestRequest("/items/{id}/release_payment", Method.PATCH);
             request.AddUrlSegment("id", itemId);
```

Comment:
so the replace url segment looks for {variable} ?

Reply:
I have been searching around, and I can't fix a single example of them using :variable, only {varable}

Reply:
aa, then it's clearly a bug! Good find!